spec/unit/application/lookup_spec.rb

```diff
@@ -145,6 +181,34 @@
       lookup.options[:explain] = true
       lookup.command_line.stubs(:args).returns(['a'])
       expect { lookup.run_command }.to output(<<-EXPLANATION).to_stdout
+Searching for 'lookup_options'
+  Merge strategy hash
+    Data Binding "hiera"
+      No such key: "lookup_options"
+    Data Provider "Hiera Data Provider, version 4"
+      ConfigurationPath "#{environmentpath}/production/hiera.yaml"
+      Data Provider "common"
+        Path "#{environmentpath}/production/data/common.yaml"
+          Original path: common
+          No such key: "lookup_options"
```

Comment:
The intention here is to provide the exact same format for the explanation regardless if it's the lookup_options or data that is being looked up. Advantage being that only one implementation is needed and only one format needs to be learned and documented. We can improve that format overall for sure, but I'm not too keen on having a separate format for the lookup_options.